usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -689,6 +689,106 @@ def set_if_found(dl_name, path, model, file):
+        },
+    },
+    'subaward': {
+        'd1': {
+            'subaward_number': 'subaward_number',
+            'prime_award_id': 'award_id',
+            'subaward_amount': 'amount',
+            'subaward_action_date': 'action_date',
+            'subaward_report_year': 'award_report_fy_year',
+            'subaward_report_month': 'award_report_fy_month',
+            'prime_awarding_agency_code': 'award__awarding_agency__toptier_agency__cgac_code',
+            'prime_awarding_agency_name': 'award__awarding_agency__toptier_agency__name',
+            'prime_awarding_sub_agency_code': 'award__awarding_agency__subtier_agency__subtier_code',
+            'prime_awarding_sub_agency_name': 'award__awarding_agency__subtier_agency__name',
+            'prime_awardee_duns': 'award__recipient__recipient_unique_id',
+            'prime_awardee_name': 'award__recipient__recipient_name',
+            'subawardee_duns': 'recipient__recipient_unique_id',
+            'subawardee_name': 'recipient__recipient_name',
+            'subawardee_address_line_1': 'recipient__location__address_line1',
+            'subawardee_city_name': 'recipient__location__city_name',
+            'subawardee_state_code': 'recipient__location__state_code',
+            'subawardee_state_name': 'recipient__location__state_name',
+            'subawardee_zip_4': 'recipient__location__zip4',
+            'subawardee_congressional_district': 'recipient__location__congressional_code',
+            'subawardee_country_code': 'recipient__location__location_country_code',
+            'subawardee_country_name': 'recipient__location__country_name',
+            'subawardee_foreign_postal_code': 'recipient__location__foreign_postal_code',
+            'subaward_primary_place_of_performance_address_line_1': 'place_of_performance__address_line1',
+            'subaward_primary_place_of_performance_city_name': 'place_of_performance__city_name',
+            'subaward_primary_place_of_performance_state_code': 'place_of_performance__state_code',
+            'subaward_primary_place_of_performance_state_name': 'place_of_performance__state_name',
+            'subaward_primary_place_of_performance_address_zip_4': 'place_of_performance__zip4',
+            'subaward_primary_place_of_performance_congressional_district': 'place_of_performance__congressional_code',
+            'subaward_primary_place_of_performance_country_code': 'place_of_performance__location_country_code',
+            'subaward_primary_place_of_performance_country_name': 'place_of_performance__country_name',
+            'subaward_description': 'description',
+            'cfda_number': 'cfda__program_number',
+            'cfda_title': 'cfda__program_title',
+            'subawardee_business_type_code': 'recipient__business_types',
+            'subawardee_business_type_description': 'recipient__business_types_description',
+            # Commented out as LegalEntity doesn't have a foreign key to LegalEntityOfficers
```

Comment:
Is adding a FK to the model something you want to change in the DB?

Another idea, if you index the columns being joined then performance-wise it might be similar to using a FK.

Reply:
Yes, I believe a discussion should happen to figure out the best way of mapping this data. The team is aware that these will be added in later after said discussion.